internal/app/coreValues/service.go

```diff
@@ -14,10 +16,11 @@ type service struct {
 }
 
 type Service interface {
-	ListCoreValues(ctx context.Context) (resp []dto.ListCoreValuesResp, err error)
-	GetCoreValue(ctx context.Context, coreValueID string) (coreValue dto.GetCoreValueResp, err error)
-	CreateCoreValue(ctx context.Context, userId int64, coreValue dto.CreateCoreValueReq) (resp dto.CreateCoreValueResp, err error)
-	UpdateCoreValue(ctx context.Context, coreValueID string, coreValue dto.UpdateQueryRequest) (resp dto.UpdateCoreValuesResp, err error)
+	ListCoreValues(ctx context.Context) (resp []dto.CoreValue, err error)
+	GetCoreValue(ctx context.Context, coreValueID string) (coreValue dto.CoreValue, err error)
+	CreateCoreValue(ctx context.Context, coreValue dto.CreateCoreValueReq) (resp dto.CoreValue, err error)
+	UpdateCoreValue(ctx context.Context, coreValueID string, coreValue dto.UpdateQueryRequest) (resp dto.CoreValue, err error)
+	ValidateParentCoreValue(ctx context.Context, coreValueID int64) (ok bool)
```

Comment:
Is this getting called from other components? if no, don't export this method.

Reply:
Done